apps/rush-lib/src/logic/yarn/YarnShrinkwrapFile.ts

```diff
@@ -1,14 +1,23 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+// eslint-disable-next-line @typescript-eslint/typedef
+const importLazy = require('import-lazy')(require);
+
 import * as os from 'os';
-import * as lockfile from '@yarnpkg/lockfile';
+// eslint-disable-next-line @typescript-eslint/typedef
+const lockfile = importLazy('@yarnpkg/lockfile');
 import { BaseShrinkwrapFile } from '../base/BaseShrinkwrapFile';
 import { FileSystem, IParsedPackageNameOrError, InternalError } from '@rushstack/node-core-library';
 import { RushConstants } from '../RushConstants';
 import { DependencySpecifier } from '../DependencySpecifier';
 import { PackageNameParsers } from '../../api/PackageNameParsers';
 
+interface IYarnLockfileParseResult {
```

Comment:
It is not necessary to define this interface.   If you simply do this...
```ts
// TODO: Convert this to "import type" after we upgrade to TypeScript 3.8
import { ParseResult } from '@yarnpkg/lockfile';
```
...then the compiler will not emit a `require('@yarnpkg/lockfile')` statement, because it is smart enough to detect that `ParseResult` is not a runtime object.